File: CodeFights/Challenge/rightmostRoundNumber.cs
/*
author: Daniel Lozano
source: CodeFights ( https://codefights.com )
problem name: Challenge > rightmostRoundNumber
problem url: https://codefights.com/challenge/npgAxzzMjLxhaXwSM
*/

using System;

namespace CodeFights
{
    class Program
    {
        static void Main()
        {   
        }

        int rightmostRoundNumber(int[] inputArray) 
		{
		    int total = -1;
		    for (int i = 0; i < inputArray.Length; i++)
		    {
		        if(inputArray[i] % 10 == 0) total = i;
		    }
		    return total;
		}
    }
}

File: CodeFights/Challenge/rightmostRoundNumber.js
/*
author: Daniel Lozano
source: CodeFights ( https://codefights.com )
problem name: Challenge > rightmostRoundNumber
problem url: https://codefights.com/challenge/npgAxzzMjLxhaXwSM
*/

function rightmostRoundNumber(inputArray) {
    for (var i = inputArray.length - 1; i >= 0; i--){
        if (inputArray[i] % 10 === 0){
            return i; 
        }
    }
    return -1;
}
